src/fetchers/repo-fetcher.ts

```diff
@@ -1,12 +1,13 @@
 // @ts-check
+import { AxiosRequestHeaders } from "axios";
 import { retryer } from "../common/retryer";
 import { MissingParamError, request } from "../common/utils";
 
 /**
- * @param {import('Axios').AxiosRequestHeaders} variables
+ * @param {AxiosRequestHeaders} variables
```

Comment:
Since we are now using TS, I was under the impression that we don't use the JSDocs syntax to specify the types in the function description (see https://tsdoc.org).

Reply:
Yeah, then.. Do you think that we should change all JSDocs to TSDocs?

Reply:
I think since we don't need the JSDocs syntax anymore, we can remove it. But maybe it is better to do this in a separate PR. Let's see what @anuraghazra suggests.

Reply:
Ok!